channel/handlers.go

```diff
@@ -2455,11 +2455,17 @@ func (server *Server) handleNewChannelBad(conn mnet.Server, reader mpacket.Reade
 func (server *Server) handleNewChannelOK(conn mnet.Server, reader mpacket.Reader) {
 	server.worldName = reader.ReadString(reader.ReadInt16())
 	server.id = reader.ReadByte()
-	server.rates.exp = reader.ReadInt16()
-	server.rates.drop = reader.ReadInt16()
-	server.rates.mesos = reader.ReadInt16()
+	// No checking errors because we've validated a couple of times
+	exp, _ := strconv.ParseFloat(reader.ReadString(reader.ReadInt16()), 32)
```

Comment:
Why are we passing the rate values as strings?

Reply:
I thought that passing floats in a packet is complicated, didn't dig in enough probably